src/driver/mod.rs

```diff
@@ -167,7 +217,9 @@ impl Ops {
 
 impl Drop for Ops {
     fn drop(&mut self) {
-        assert!(self.lifecycle.is_empty());
-        assert!(self.completions.is_empty());
+        assert!(self
```

Comment:
Usually these are transitioned to the ignored state and then gotten rid of on completion because the runtime drops first (dropping the tasks and futures on it), but that won't happen if `std::process::exit` is being invoked, so we need a sound thing to do in the case where we are dropped but not the runtime.